Cargo.toml

```diff
@@ -73,11 +77,15 @@ surfman-chains-api = "0.2"
 thin-slice = "0.1.0"
 time = "0.1.41"
 tokio = "1"
+tokio-rustls = "0.24"
+tungstenite = "0.19"
```

Comment:
I suppose this is using older tungstenite to match async-tungstenite, and using older webpki-roots to match async-tungstenite/tungstenite? Have you looked into using tokio-tungstenite instead of async-tungstenite? It seems to keep up better with tungstenite releases at least (though both use an old version of webpki-roots for now).

Reply:
This is a good idea, though perhaps we should tackle upgrading tungstenite and switching to async-tunstenite in a followup change as it's a bit unrelated.

Reply:
Hmm, it looks like tokio-tungstenite also provides a `Connector` API that lets you provide your own `rustls::ClientConfig`. Using that would have the benefit that you no longer depend on the rustls choices/dependency versions that tokio-tungstenite comes with, so might be a good way forward (unfortunately the tokio-tungstenite Cargo features don't expose this option in a public way right now).